tests/lib/nested.sh

```diff
@@ -682,6 +682,34 @@ nested_configure_cloud_init_on_core20_vm() {
     kpartx -d "$IMAGE"
 }
 
+nested_save_serial_log() {
+    if [ -f "${NESTED_LOGS_DIR}/serial.log" ]; then
+        for i in $(seq 50); do
```

Comment:
This should be `seq 1 9`, otherwise sorting done by `ls` is not so obvious anymore, eg, `1, 10, 2, 3, ..`.

Reply:
sorry, did that change but I lost that.